docs/modules/ROOT/pages/explanations/commodore-packages.adoc

```diff
@@ -0,0 +1,150 @@
+= Commodore Package Best Practices
+
+Commodore Configuration Packages provide a way of reusing configuration presets for one or more components.
+
+In this document we provide a few best practices for writing packages.
+While adhering to these practices isn't mandatory, it makes using packages more consistent and allows Syn tooling, such as the https://github.com/projectsyn/commodore-renovate[Commodores Renovate extension], to interact with them.
+
+== Structure
+
+The structure of Commodore Packages is a lot less rigid than for Commodore Components.
+The only constraint is that they need to provide one or more reclass classes in the form of YAML files.
+
+However, to have some consistent feel when using and working on packages we recommend a basic structure:
```

Comment:
I found your sentence kinda hard to read, but I'm not sure my suggestion is better.

```suggestion
However, to give packages a consistent look and feel for users and developers we recommend a basic structure:
```